README.md

```diff
@@ -433,6 +433,20 @@ const { output } = await container.exec(["curl", `http://host.testcontainers.int
 assert(output === "hello world");
 ```
 
+Specifying an exact host port to bind to (not recommended, Testcontainers will automatically find an available port):
+
+```javascript
+const { GenericContainer, Network } = require("testcontainers");
```

Comment:
could you please remove the network part from this example?

Reply:
My bad, fixed in 6360c4b.